sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSub/src/Microsoft.Azure.WebJobs.Extensions.WebPubSub.csproj

```diff
@@ -26,9 +26,9 @@
     <!--
 	TODO: Changing to a PackageReference since we only want to depend on the already GA'ed version of WebPubSub.  Change back after release.
 	<ProjectReference Include="..\..\Azure.Messaging.WebPubSub\src\Azure.Messaging.WebPubSub.csproj" />
-    <ProjectReference Include="..\..\Microsoft.Azure.WebPubSub.Common\src\Microsoft.Azure.WebPubSub.Common.csproj" />
+    <PackageReference Include="Microsoft.Azure.WebPubSub.Common" />
 	-->
     <PackageReference Include="Azure.Messaging.WebPubSub" />
-    <PackageReference Include="Microsoft.Azure.WebPubSub.Common" />
+    <ProjectReference Include="..\..\Microsoft.Azure.WebPubSub.Common\src\Microsoft.Azure.WebPubSub.Common.csproj" />
```

Comment:
Based on the comment above should this not be a package reference?

Reply:
We'd  like to release the 2 packages together. Should they use project reference? I'm wondering package reference makes it release separately. Would you help confirm?
* Microsoft.Azure.WebPubSub.Common
* Microsoft.Azure.WebJobs.Extensions.WebPubSub

Reply:
@pallavit Would you help confirm?

Reply:
Yes package reference means you are picking things up from the nuget instead of building it from the source. Looking at the ci.yml file I can see common and webpubsub package being built separately.

Reply:
So if we'd like to release them together instead of one by one, shall we change back to project reference like this? If this is the correct way, would you help approve API change so we can release it?

Reply:
You will have to do package reference instead of project reference and then release both the SDKs via the CI pipeline. Sorry if I was not clear.

Reply:
Package reference makes `Microsoft.Azure.WebJobs.Extensions.WebPubSub` reference `Microsoft.Azure.WebPubSub.Common` v1.1.0 instead of the releasing one v1.2.0. I'm wondering that use package reference would require another PR to update package version to v1.2.0 so we can release `Microsoft.Azure.WebJobs.Extensions.WebPubSub`?

`Microsoft.Azure.WebJobs.Extensions.WebPubSub` has dependency on `Microsoft.Azure.WebPubSub.Common` and we'd like to make the change to both of them.

To save the time, I'm changing to package reference per your guidance. If release separately is current CI's requirement, I'm good with it, too.

Reply:
You'd need to release the dependencies first then update _eng/Packages.Data.props_ to pick them up from nuget. This is why, for example, we release any necessary changes to Azure.Core - on which all our other track SDKs depend - first before we release any other track 2 SDKs that depend on new features in Azure.Core.

Reply:
I see. Thank @heaths for clarification. Is current version good to release `Microsoft.Azure.WebPubSub.Common` first? I can update _Packages.Data.props_ after then.